### ISSUE TITLE
New dependencies

### ISSUE THREAD

**Issue by muratmaga** [SlicerMorph/SlicerMorph#355]
'Surface Markups' and 'Sandbox' extensions need to be added as automatically installed dependencies.

**Comment by smrolfe** [comment]
SurfaceMarkup is installed at runtime of the PlaceLandmarkGrid module. Sandbox is currently a recommended, manually installed extension since it is not a true dependency. We could add an automatic runtime install?

**Comment by amm554** [comment]
Yes, please make an automatic install. it is necessary for cloud workshops so that it is installed with slicermorph

**Comment by amm554** [comment]
No, runtime dependencies. Please install straight up.

**Comment by smrolfe** [comment]
Got it, thanks!

**Comment by smrolfe** [comment]
In progress here: https://github.com/Slicer/ExtensionsIndex/pull/2115

**Comment by muratmaga** [comment]
done
